src/main/java/org/spongepowered/api/effect/sound/SoundType.java

```diff
@@ -33,4 +36,50 @@
 @CatalogedBy(SoundTypes.class)
 public interface SoundType extends CatalogType {
 
+    /**
+     * Creates a new {@link Builder} for building SoundTypes.
+     *
+     * @return A new builder
+     */
+    static Builder builder() {
+        return Sponge.getRegistry().createBuilder(SoundType.Builder.class);
+    }
+
+    /**
+     * Creates a <i>new</i>SoundType from the given ID. To fetch existing types,
+     * use {@link GameRegistry#getType(Class, String)}.
+     *
+     * @param id ID of the sound
+     * @return A new SoundType object
+     */
+    static SoundType of(String id) {
+        return builder().id(id).build();
+    }
+
+    /**
+     * Builds a SoundType, primarily for sending custom sounds to the client.
+     */
+    interface Builder extends ResettableBuilder<SoundType, Builder> {
+
+        /**
+         * Resource location for the sound. If no domain (indicated by the
+         * string before ':') is present, the default "minecraft" domain will
+         * be used.
+         *
+         * @param id ID of the sound
+         * @return This builder, for chaining
+         */
+        Builder id(String id);
```

Comment:
@ST-DDT All builders creating cataloged types should take the id as a parameter to the build method (and looking around a bit all the ones I found either did).